src/wp-includes/formatting.php

```diff
@@ -2306,6 +2306,10 @@ function sanitize_title_with_dashes( $title, $raw_title = '', $context = 'displa
 	$title = preg_replace( '|-+|', '-', $title );
 	$title = trim( $title, '-' );
 
+	if ( seems_utf8( $title ) ) {
+		$title = substr( $title, 0, 200 );
```

Comment:
Just wondering: should this be 200 bytes or 200 chars ?
If the text is utf8, using `substr()` may cut the string mid-character. Would `mb_substr()` possibly be more appropriate ?
(and yes, this can be safely used, the function is polyfilled in case MbString is not available)

Reply:
Hmm... on second look, looks like it is stripped down to alphanumeric. In which case it leaves me wondering why the cut-off is only done for utf8 ? I must be missing something.

Reply:
@jrfnl I was wondering that myself about the cut-off only being done for utf8.

Reply:
I have a suspicion this needs further investigation and discussion.